scripts/increment-cargo-version.sh

```diff
@@ -45,8 +45,10 @@ MAJOR=0
 MINOR=0
 PATCH=0
 SPECIAL=""
+WORKSPACE_CARGO_TOML="Cargo.toml"
```

Comment:
The script had updates so this should actually be reviewed. I was able to run `scripts/increment-cargo-version.sh check` and `scripts/increment-cargo-version.sh minor` without issue